make/test.mk

```diff
@@ -30,10 +30,12 @@ endif
 
 E2E_TEST_EXECUTION ?= true
 
+
 ifeq ($(IS_OSD),true)
```

Comment:
this has to be 
```suggestion
ifneq ($(IS_OSD),true)
```

Reply:
yes i was thinking about the same.. but since it was also being used here.. https://github.com/codeready-toolchain/toolchain-e2e/blob/a5241b5effc70aec391331d8cd44ccffdf801479/make/test.mk#L80
i wasnt still trying to check that it doesnt wont change the usage at the above place..

Reply:
the line only passes the current value of the variable to the called target - to preserve the value. Or what issue do you see with this? If you didn't fix the if statement, then it would work in the opposite way than it is supposed to.

Anyway, what kind of concern did you see with applying the change?

Reply:
now that we have changed the whole param to `--lets-encrypt=false` instead of just `--lets-encrypt` i was just trying to understand if/how that would still work along with the boolean value we are passing.. 

i have already applied the change .

Reply:
This looks fine, even if it relies on the fact that the ksctl command has lets-encrypt set to true by default and it doesn't pass explicitly true. But it's a minor thing.